code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -58,6 +58,11 @@ var/list/diona_banned_languages = list(
 		sprint_speed_factor = 0.75
 		sprint_cost_factor = 0
 
+/mob/living/carbon/alien/diona/death()
+	if(client)
+		to_chat(switch_to_gestalt(), span("warning", "You feel tremendous pain as you loose connection to your detached nymph, there is no way to bring it back anymore."))
```

Comment:
`switch_to_gestalt()` doesn't return any valid values.

```dm
/mob/living/carbon/alien/diona/proc/switch_to_gestalt()
	set name = "Switch to Gestalt"
	set desc = "Allows you to switch control back to your parent Gestalt."
	set category = "Abilities"

	if(!gestalt)
		to_chat(src, span("warning", "You have no Gestalt!"))
	else if(gestalt.stat == DEAD)
		to_chat(src, span("danger", "Your Gestlat is not responding! Something could have happened to it!"))
	else
		gestalt.key = key
```

Reply:
Yes it does, line 594 of changed files. Your probably took this code from master